docs/cni-plugin.md

```diff
@@ -0,0 +1,91 @@
+# Open vSwitch CNI Plugin
+
+## Overview
+
+With ovs plugin, containers (on the same host) are plugged into an Open vSwitch
+bridge (virtual switch) that resides in the host network namespace. It's host
+adminitrator's responsibility to create such bridge and optinally connect it to
+broader network, be it using L2 directly, NAT or an overlay. The containers
+receive one end of the veth pair with the other end connected to the bridge.
+
+## Example Configuration
+
+```json
+{
+    "name": "mynet",
+    "type": "ovs",
+    "bridge": "mynet0",
+    "vlan": 100
+}
+```
+
+## Network Configuration Reference
+
+* `name` (string, required): the name of the network.
+* `type` (string, required): "ovs".
+* `bridge` (string, required): name of the bridge to use.
+* `vlan` (integer, optional): VLAN ID of attached port. Opened for all if not
+   specified.
+
+## Manual Testing
+
+```shell
+# Build the binary
+make build-plugin
+
+# Create a new namespace
+ip netns add ns1
+
+# Create OVS bridge on the host
```

Comment:
add here how to install ovs or maybe just a note that ovs needs to be installed and running